.changeset/.PREAMBLE.md

```diff
@@ -0,0 +1,8 @@
+This version of skuba should not be a large upgrade effort for most use-cases, but does contains some large changes:
+
+- Major upgrades to the linting and patching systems, which may result in some code changes on version upgrades.
+  To make upgrades easy now and going forward, we recommend setting up [GitHub autofixes](https://seek-oss.github.io/skuba/docs/deep-dives/github.html#github-autofixes).
+- New projects will now be initialised with `pnpm`.
+  A future release of `skuba` will likely transition existing projects to `pnpm` as well.
```

Comment:
Still paranoid about changeset formatting, and I figured we should avoid saying "major" on a "minor" release 🙂

```suggestion
This version of skuba should not require significant upgrade effort for most projects, but it does contain some notable changes:

- Internal linting and patching have been overhauled to streamline code generation on version upgrades.

  To make upgrades easy now and going forward, we recommend setting up [GitHub autofixes](https://seek-oss.github.io/skuba/docs/deep-dives/github.html#github-autofixes).

- New projects will now be initialised with pnpm.

  A future release of skuba may transition existing projects to pnpm.
```